coalib/output/ConsoleInteraction.py

```diff
@@ -166,6 +168,13 @@ def acquire_actions_and_apply(console_printer,
     :param cli_actions:     The list of cli actions available.
     """
     cli_actions = CLI_ACTIONS if cli_actions is None else cli_actions
+    if result.alternate_diffs is not None:
+        name = 'AlternatePatchAction {}'
+        count = 0
+        for alternate_diff in result.alternate_diffs:
+            count += 1
+            cli_actions += (AlternatePatchAction(alternate_diff,
+                                                 name.format(str(count))),)
```

Comment:
Since there are multiple instances of the same class, it causes trouble recognizing which is which when user chooses one of this. That is why `AlternatePatchAction` needs it own `get_metadata` function also.